examples/apps/cra-demo/README.md

```diff
@@ -36,24 +35,35 @@ npx create-react-app my-app-name
 cd my-app-name
 ```
 
-\* Don't start the React application yet. We'll do that in step #6.
+### 1.a Start the servers
```

Comment:
Now the demo should build/run from the getgo. No waiting till the end to run it

Reply:
thoughts on something like "Start the servers and app" instead of just servers?

Reply:
```suggestion
### 1.a Start the app
```

yeah, guess only 1 is a server. Point is to start and see the application. I'm good with just 'start the app'